frontend/src/store/featureStore.ts

```diff
@@ -0,0 +1,19 @@
+import { defineStore } from 'pinia';
+import { computed } from 'vue';
+
+import useConfigStore from '@/store/configStore';
+import { isTruthy } from '@/utils/utils';
+
+export const useFeatureStore = defineStore('feature', () => {
+  // Getters
+  const getters = {
+    isPeachEnabled: computed(() => isTruthy(useConfigStore().getConfig.features.peach))
```

Comment:
Future consideration, if we start having many feature flags (not likely anytime soon) we can do a single getter like `hasFeature(SOME_FEATURE_KEY)` similar to the `canNavigate` in `authzStore`